spyderlib/requirements.py

```diff
@@ -39,7 +39,7 @@ def check_qt():
     try:
         import qtpy
         package_name, required_ver = qt_infos[qtpy.API]
-        actual_ver = qtpy.__version__
+        actual_ver = qtpy.PYQT_VERSION

```

Comment:
We are not allowing PySide in `qt_infos` so it's really unsupported at this time. But that's for another PR :-)